repository/src/main/java/org/apache/atlas/tasks/TaskManagement.java

```diff
@@ -251,6 +261,16 @@ private void startInternal() {
             return;
         }
 
+        String currentHostName = System.getenv(HOST_NAME);
+        String taskExecutionPodName = AtlasConfiguration.TASK_EXECUTION_POD_HOST_NAME.getString();
+
+        if (!taskExecutionPodName.isEmpty() && currentHostName != null) {
```

Comment:
I suggest using StringUtils.isEmpty, StringUtils.isNotEmpty, it does both null & empty check